.readthedocs.yml

```
@@ -9,6 +9,13 @@ build:
   tools:
     python: "3.9"
     nodejs: "19"
```

```suggestion
  tools:
    python: "3.9"
```

I know this is the same in kedro-org/kedro but maybe it's not even needed? I introduced it to generate the Mermaid diagrams, but after a while we ended up making them static https://github.com/kedro-org/kedro/pull/3449